api/Models/UpdateEventInput.go

```diff
@@ -0,0 +1,11 @@
+package Models
+
+import "time"
+
+type UpdateEventInput struct {
```

Comment:
What is this model for?

Reply:
It is a separate struct to represent the fields that can be updated using the PUT endpoint (Everything can be updated except the id)